packages/client/base/src/types/hosted/index.ts

```diff
@@ -43,8 +43,8 @@ export type CrossmintPayButtonProps = BaseButtonProps & {
 
 export type CheckoutProps = {
     experimental?: boolean;
-    display: "same-tab" | "new-tab" | "popup";
-    delivery: "custodial" | "non-custodial" | "all";
+    display?: "same-tab" | "new-tab" | "popup";
+    delivery?: "custodial" | "non-custodial" | "all";
```

Comment:
why is this changing?

Reply:
@pgarciaegido this extra mille: https://linear.app/crossmint/issue/PAY-2484/checkout-form-in-react-not-working-when-all-checkoutprops-properties

 These are actually optionals and have default values.